CHANGELOG.md

```diff
@@ -21,8 +21,9 @@
 
 ### Download
 
-- Replace `--tower` with `--platform`. The former will remain for backwards compatability for now but will be removed in a future release.
+- Replace `--tower` with `--platform`. The former will remain for backwards compatability for now but will be removed in a future release. ([#2853](https://github.com/nf-core/tools/pull/2853))
 - Better error message when GITHUB_TOKEN exists but is wrong/outdated
+- New `-a` / `--additional-tag` argument to add custom tags during a pipeline download ([#2938](https://github.com/nf-core/tools/pull/2938))
```

Comment:
tbh not a big fan to use `-a` for something other then `--all` 🙂 how about just going for `--tag`?

Reply:
Notwithstanding, it is so forthcoming of Seqera to rename their web application just to free up the `-t` short-form flag for the future 😝...

Reply:
👉🏻  https://github.com/nf-core/tools/pull/2938#discussion_r1589611191